src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -2637,12 +2637,12 @@ private static CryptoPermission getConfiguredPermission(
      * policy files, please see the Cryptographic defaults and limitations in
      * the {@extLink security_guide_jdk_providers JDK Providers Documentation}.
      *
-     * @param transformation the cipher transformation.
-     * @return the maximum key length in bits or Integer.MAX_VALUE.
-     * @exception NullPointerException if {@code transformation} is null.
+     * @param transformation the cipher transformation
+     * @return the maximum key length in bits or Integer.MAX_VALUE
```

Comment:
Integer.MAX_VALUE should be inside a `@code` tag.

Reply:
Ok.